x-pack/plugins/cases/common/api/cases/configure.ts

```diff
@@ -6,26 +6,26 @@
  */
 
 import * as rt from 'io-ts';
+import { CaseConnectorRt } from '../connectors/connector';
```

Comment:
I was running into weird circular dependencies so importing the files directly to fix it.

Reply:
Yes @js-jankisalvi had the same issue in her PR!